scenarios/search-menu.json

```diff
@@ -108,15 +108,16 @@
       "label": "Full-text Search - result",
       "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1",
       "hideSelectors": ["#mainTabContainer"],
-      "onReadyScript": "search/full-text-result.js"
+      "onReadyScript": "search/full-text-result-activity.js"
     },
     {
       "label": "Full text Search - Contact - Results",
       "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1&force=1&table=Contact"
     },
     {
       "label": "Full text Search - Activities - Results",
-      "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1&force=1&table=Activity"
+      "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1",
+      "onReadyScript": "search/structure-table.js"
```

Comment:
`structure-table` doesn't really say what the script does, is it another generic `show.js` script?

Reply:
Nope! Let me rename it for better.